components/list/list-item-generic-layout.js

```diff
@@ -144,12 +156,8 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 	}
 
 	firstUpdated() {
-		this.addEventListener('keydown', this._handleKeydown.bind(this));
+		this.addEventListener('keydown', this._onKeydown.bind(this));
 		this.addEventListener('focusin', this._setFocusInfo.bind(this));
-		this.addEventListener('d2l-dropdown-open', this._onDropdownOpen.bind(this));
-		this.addEventListener('d2l-dropdown-close', this._onDropdownClose.bind(this));
-		this.addEventListener('d2l-tooltip-show', this._onTooltipShow.bind(this));
-		this.addEventListener('d2l-tooltip-hide', this._onTooltipHide.bind(this));
```

Comment:
These listeners have been moved to the `list-item`. The `postition: relative` on the `list-item` prevents this fix from working. I'm considering putting in the `ListItemRoleMixin` and renaming that mixin back to `ListItemMixin`

Reply:
So in the future if we find we need to make custom list items in HM. I want `ListItemMixin` to be extendable such that it the same as `list-item` but instead of slots there is a render method with `_renderListItem({differentSlots})` They will also need to extend the style too. So that is why I ask for the rename.

Reply:
Yes, I'd be happy with it being named to something like `listItemRequiredMixin` Again, `listItemMixin` simple name to be style based in the future. So when we have other styles they can also be mixins.